src/cross/widgets/MiscUtil.h

```diff
@@ -12,7 +12,7 @@ QByteArray & ByteReverse(QByteArray & array);
 QByteArray ByteReverse(QByteArray && array);
 bool SimpleInputBox(QWidget* parent, const QString & title, QString defaultValue, QString & output, const QString & placeholderText, const QIcon* icon = nullptr);
 bool SimpleChoiceBox(QWidget* parent, const QString & title, QString defaultValue, const QStringList & choices, QString & output, bool editable, const QString & placeholderText, const QIcon* icon = nullptr, int minimumContentsLength = -1);
-void SimpleErrorBox(QWidget* parent, const QString & title, const QString & text);
+void SimpleErrorBox(QWidget* parent, const QString & title, const QString & text, const QString & hideConfigCategory = nullptr, const QString & hideConfigId = nullptr);

```

Comment:
I think the interface should be simplified to only have `hideConfigId` and always use the `Gui` section

Reply:
Also it should be `= {}` not `= nullptr`